include/odp/api/spec/classification.h

```diff
@@ -581,6 +583,9 @@ typedef enum {
 	/** Destination SCTP port (val_sz = 2) */
 	ODP_PMR_SCTP_DPORT,
 
+	/** GTPU Tunnel endpoint identifier  (val_sz = 4), with UDP port 2152 */
+	ODP_PMR_GTPU_TEID,
```

Comment:
More specifically, the UDP port is UDP _destination_ port.

I still wonder what the exact matching rule is. Is this equivalent to matching packets with UDP destination port 2152 (implying IP proto == UDP) and matching the 4 bytes at offset 4 of UDP payload with the specified TEID value? Or does this parse the GTP header more so that not all such packets match (e.g. by checking GTP version field etc)?

What I am looking for is an unambiguous definition that tells if a packet containing an arbitrary bit pattern matches or not.